tests/bluetooth/tester/src/btp_bap_stream.h

```diff
@@ -0,0 +1,216 @@
+uint8_t btp_bap_broadcast_sink_release(const void *cmd, uint16_t cmd_len,
+				       void *rsp, uint16_t *rsp_len);
+uint8_t btp_bap_broadcast_scan_start(const void *cmd, uint16_t cmd_len,
+				     void *rsp, uint16_t *rsp_len);
+uint8_t btp_bap_broadcast_scan_stop(const void *cmd, uint16_t cmd_len,
+				    void *rsp, uint16_t *rsp_len);
+uint8_t btp_bap_broadcast_sink_sync(const void *cmd, uint16_t cmd_len,
+				    void *rsp, uint16_t *rsp_len);
+uint8_t btp_bap_broadcast_sink_stop(const void *cmd, uint16_t cmd_len,
+				    void *rsp, uint16_t *rsp_len);
+uint8_t btp_bap_broadcast_sink_bis_sync(const void *cmd, uint16_t cmd_len,
+					void *rsp, uint16_t *rsp_len);
+uint8_t btp_bap_broadcast_discover_scan_delegators(const void *cmd, uint16_t cmd_len,
+						   void *rsp, uint16_t *rsp_len);
+uint8_t btp_bap_broadcast_assistant_scan_start(const void *cmd, uint16_t cmd_len,
+					       void *rsp, uint16_t *rsp_len);
+uint8_t btp_bap_broadcast_assistant_scan_stop(const void *cmd, uint16_t cmd_len,
+					      void *rsp, uint16_t *rsp_len);
+uint8_t btp_bap_broadcast_assistant_add_src(const void *cmd, uint16_t cmd_len,
+					    void *rsp, uint16_t *rsp_len);
+uint8_t btp_bap_broadcast_assistant_remove_src(const void *cmd, uint16_t cmd_len,
+					       void *rsp, uint16_t *rsp_len);
+uint8_t btp_bap_broadcast_assistant_modify_src(const void *cmd, uint16_t cmd_len,
+					       void *rsp, uint16_t *rsp_len);
+uint8_t btp_bap_broadcast_assistant_set_broadcast_code(const void *cmd, uint16_t cmd_len,
+						       void *rsp, uint16_t *rsp_len);
+uint8_t btp_bap_broadcast_assistant_send_past(const void *cmd, uint16_t cmd_len,
+					      void *rsp, uint16_t *rsp_len);
+
+uint8_t btp_ascs_configure_codec(const void *cmd, uint16_t cmd_len, void *rsp, uint16_t *rsp_len);
+uint8_t btp_ascs_configure_qos(const void *cmd, uint16_t cmd_len, void *rsp, uint16_t *rsp_len);
+uint8_t btp_ascs_enable(const void *cmd, uint16_t cmd_len, void *rsp, uint16_t *rsp_len);
+uint8_t btp_ascs_receiver_start_ready(const void *cmd, uint16_t cmd_len,
+				      void *rsp, uint16_t *rsp_len);
+uint8_t btp_ascs_receiver_stop_ready(const void *cmd, uint16_t cmd_len,
+				     void *rsp, uint16_t *rsp_len);
+uint8_t btp_ascs_disable(const void *cmd, uint16_t cmd_len, void *rsp, uint16_t *rsp_len);
+uint8_t btp_ascs_release(const void *cmd, uint16_t cmd_len, void *rsp, uint16_t *rsp_len);
+uint8_t btp_ascs_update_metadata(const void *cmd, uint16_t cmd_len, void *rsp, uint16_t *rsp_len);
+uint8_t btp_ascs_add_ase_to_cis(const void *cmd, uint16_t cmd_len, void *rsp, uint16_t *rsp_len);
+uint8_t btp_ascs_preconfigure_qos(const void *cmd, uint16_t cmd_len, void *rsp, uint16_t *rsp_len);
+
+void btp_send_stream_received_ev(struct bt_conn *conn, struct bt_bap_ep *ep,
+				 uint8_t data_len, uint8_t *data);
+void btp_send_bis_stream_received_ev(const bt_addr_le_t *address, uint32_t broadcast_id,
+				     uint8_t bis_id, uint8_t data_len, uint8_t *data);
```

Comment:
The name of this file, `btp_bap_stream.h` and the names of these functions do not seem to match. The functions do not seem to be fully coupled for streams.

Reply:
I've splitted the `btp_bap_stream.h` into `btp_bap_audio_stream.h`, `btp_bap_unicast.h` and `btp_bap_broadcast.h`